examples/stable-video-diffusion/tests/test_svd.py

```diff
@@ -0,0 +1,30 @@
+from typing import List, Iterable
+
+from PIL import Image
+
+from nos.client import Client
+
+TEST_IMAGE_LINK = "https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/svd/rocket.png?download=true"
+
+def test_svd():
+    client = Client("[::]:50051")
+    assert client is not None
+    assert client.WaitForServer()
+    assert client.IsHealthy()
+
+    model_id = "stable-video-diffusion"
+    models: List[str] = client.ListModels()
+    assert model_id in models
+
+    model = client.Module(model_id)
+    assert model is not None
+    assert model.GetModelInfo() is not None
+    print(f"Test [model={model_id}]")
+
+    response: Iterable[Image.Image] = model.image2video(image=TEST_IMAGE_LINK, _stream=True)
```

Comment:
Same comment as above, would be good to pass in Image.Image instead of a link. But otherwise, looks great!

Reply:
Got it, changed.